lib/pltr/v2/helpers/hierarchy.js

```diff
@@ -27,10 +27,11 @@ const noneIsTransparent = (borderStyle, borderColor) => {
 export const hierarchyToStyles = (
   { level, textColor, textSize, borderColor, borderStyle, backgroundColor },
   timelineSize,
-  hovering
+  hovering,
+  isDarkMode
 ) => ({
   ...{
-    color: nullIfNone(textColor),
+    color: nullIfNone(textColor === '#0B1117' && isDarkMode ? '#eee' : textColor),
```

Comment:
Here's where I think Plottr should do all the checking.
If the colour is `none` and we're in light mode, let's use the dark colour, and we can use the light colour in dark mode.

Finally, can we put those two colours into the colours helper file in `pltr` (`lib/pltr/v2/helpers/colors.js`) and then export a function from there called `textColor` whose job it is to produce the correct text colour given the single argument `isDarkMode`.

Reply:
You should also check out Colin's PR: https://github.com/cameronsutter/plottr_electron/pull/621

I think we should use his strategy. That might solve this issue as well. Or we might be able to solve this issue using his PR